app/scripts/directives/bindService.js

```diff
@@ -178,10 +180,13 @@
       });
 
       if (ctrl.target.kind === 'Instance') {
-        ctrl.bindType = "secret-only";
-        ctrl.appToBind = null;
-        ctrl.serviceToBind = ctrl.target.metadata.name;
-        loadApplications();
+        ProjectsService.get(ctrl.target.metadata.namespace).then(_.spread(function(project) {
```

Comment:
This is going to cause another request for the project when we already have the project loaded